src/pages/[platform]/build-a-backend/auth/manage-users/manage-webauthn-credentials/index.mdx

```diff
@@ -28,3 +28,131 @@ export function getStaticProps() {
   };
 }
 
+<InlineFilter filters={["react-native"]}>
+<Callout warning>
+
+WebAuthn registration and authentication are not currently supported on React Native, other passwordless features are fully supported.
+
+</Callout>
+</InlineFilter>
+
+Amplify Auth enables your users to associate, keep track of, and delete passkeys.
+
+## Associate a passkey
+
+Note that users must be authenticated to register a passkey. That also means users cannot create a passkey during sign up; consequently, they must have at least one other first factor authentication mechanism associated with their account to use WebAuthn.
+
+You can associate a passkey using the following API:
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+```ts
+import { associateWebAuthnCredential} from 'aws-amplify/auth';
+
+await associateWebAuthnCredential();
+```
+
+The associate passkey API has no inputs, and it does not return a value.
```

Comment:
Same as above this is an implementation detail and may not need explicit call out.